pkg/opsmanager/api_set_project_tags.go

```diff
@@ -0,0 +1,30 @@
+package opsmanager
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/mongodb-labs/pcgc/pkg/useful"
+)
+
+func (client opsManagerClient) SetProjectTags(projectID string, tags []string) (ProjectResponse, error) {
+	var result ProjectResponse
+
+	url := client.resolver.Of("/groups/%s", projectID)
+
+	request := struct {
+		Tags []string `json:"tags,omitempty"`
+	}{tags}
+
+	body, err := json.Marshal(request)
+	if err != nil {
+		return result, err
```

Comment:
is this error recoverable? My guess is not, since if you can't marshall the request, something might have gone wrong.  

I'd also use PanicOnUnrecoverableError here.

Reply:
I've seen this in a few places; if you agree with the above, let's change them all.

Reply:
Right, I've missed that. I'll change it in accordance with the rest of the code

Reply:
After taking another look, this seems to be the way most `json.Marshal` calls are handled currently, for example in `api_create_first_user.go`

Reply:
It's not a big deal, to be honest. At this point in the project, it's hard to tell which calls can be recovered from (we don't actually recover from any.)

Feel free to leave it as it is.